src/python/WMCore/WorkQueue/WorkQueueBackend.py

```diff
@@ -385,7 +385,7 @@ def availableWork(self, thresholds, siteJobCounts, teams = None, wfs = None):
                 elements.append(element)
                 if site not in siteJobCounts:
                     siteJobCounts[site] = {}
-                siteJobCounts[site][prio] = siteJobCounts[site].setdefault(prio, 0) + element['Jobs']
+                siteJobCounts[site][prio] = siteJobCounts[site].setdefault(prio, 0) + element['Jobs']*(1+element.get('BlowupFactor', 0.0))
```

Comment:
How about setting the default value to 1.0 instead? Such that we get rid of the 1+ in this line